content-services/7.4/tutorial/sso/ldap.md

```diff
@@ -0,0 +1,296 @@
+| identity-service.public-client | The adapter will not send credentials for the client to Keycloak if this is set to true, for example `true`. |
+| identity-service.credentials.secret | The secret key for this client if the access type is not set to public. |
+
+## Step 7: Configure Alfresco Process Services
+
+Alfresco Process Services (APS) has two sets of properties that need to be configured to setup SSO. One set synchronizes APS with an LDAP directory and the other set configures the connection with Keycloak.
+
+1. Configuration for LDAP synchronization can be achieved manually for WAR file deployments using the `activiti-ldap-properties` file or reference an external file for Docker and Kubernetes deployments:
+
+    | Property | Description |
+    | -------- | ----------- |
+    | ldap.authentication.enabled | Sets whether LDAP authentication is enabled. This needs to be `false` as LDAP is only being used for user synchronization, for example `false`. |
+    | ldap.authentication.java.naming.provider.url | The URL of the LDAP instance, for example `ldaps://ldap.example.com:636`. |
+    | ldap.synchronization.java.naming.security.principal | The user used to access the LDAP directory to perform the synchronization, for example `uid=admin,ou=system`. |
+    | ldap.synchronization.java.naming.security.credentials | The password for the user set in `ldap.synchronization.java.naming.security.principal`, for example `secret`. |
+    | ldap.synchronization.full.enabled | Sets whether full LDAP synchronization is enabled or not, for example `true`. |
+    | ldap.synchronization.full.cronExpression | The cron expression describing how often the full synchronization should run, for example `0 0 0 * * ?`. |
+    | ldap.synchronization.differential.enabled | Sets whether differential LDAP synchronization is enabled or not, for example `true`. |
+    | ldap.synchronization.differential.cronExpression | The cron expression describing how often the differential synchronization should run, for example `0 0 */4 * * ?`. |
+    | ldap.synchronization.userSearchBase | The section of the LDAP directory to restrict user synchronization to, for example `ou=users,dc=alfresco,dc=com`. |
+    | ldap.synchronization.groupSearchBase | The section of the LDAP directory to restrict group synchronization to, for example `ou=groups,dc=alfresco,dc=com`. |
+
+2. Configuration with the Alfresco Process Services can be achieved manually for WAR file deployments using the `activiti-identity-service.properties` or reference an external file for Docker and Kubernetes deployments:
+
+    | Property | Description |
+    | -------- | ----------- |
+    | keycloak.enabled | Sets whether Process Services will use Keycloak to authenticate against, for example `true`. |
```

Comment:
For which version of APS is this document for? These properties will work for APS 2.x and will not for APS 24.x

Reply:
Hi @balsarori, this is for ACS 7.4 which is compatible with APS 2.x. For ACS 23.x (which is compatible with APS 24.x) these properties are different: mainly "keycloak" is replaced with "activiti.identity-service". Is it correct?